src/applications/personalization/dashboard/containers/DashboardApp.jsx

```diff
@@ -481,26 +437,7 @@ class DashboardApp extends React.Component {
       </div>
     );
 
-    return (
```

Comment:
@bshyong is this what you had in mind when setting up the `DashboardAppWrapper` component?